lib/datadog/profiling/collectors/cpu_and_wall_time_worker.rb

```diff
@@ -37,12 +38,17 @@ def initialize(
             )
           end
 
+          if no_signals_workaround_enabled
+            Datadog.logger.debug('Profiling no signals workaround is in use. Profiling data will have lower precision.')
+          end
```

Comment:
It looks like when initializing the component `lib/datadog/profiling/component.rb` we log enough information about what would happen if we use `no_signals_workaround`

Do we still need this log line?

Reply:
Good catch. I initially thought this may be useful for debugging, but yeah thinking about it again let's just remove it 09fef75491022bcee0126b84297fe5253df949a0 .